src/sagas/projects.js

```diff
@@ -131,6 +171,10 @@ export default function* projects() {
       'UPDATE_PROJECT_SOURCE',
       'UPDATE_PROJECT_INSTRUCTIONS',
     ], updateProjectSource),
+    throttle(100, [
+      'UPDATE_PROJECT_SOURCE',
```

Comment:
For `UPDATE_PROJECT_SOURCE`, wouldn’t it be more efficient to call a different saga that only re-parses and calculates the selector locations for that language?